_includes/setup/editor-setup.md

```diff
@@ -0,0 +1,7 @@
+## Editor setup
+
+Using our command-line tools, you can use any editor to develop Flutter applications.
```

Comment:
Should we mention that the command line tool is called `flutter` and you can learn more about it by typing `flutter help` in a terminal?